### PR TITLE
Broker storage

### PR DESCRIPTION
**Pull request by chemicL**
Handling the new broker storage with Kafka 0.8.2 with backwards compatibility.
https://github.com/quantifind/KafkaOffsetMonitor/issues/29

### CODE REVIEW

File: src/main/scala/com/quantifind/kafka/OffsetGetter.scala [hunk 1]
```diff
@@ -119,7 +150,7 @@ class OffsetGetter(zkClient: ZkClient) extends Logging {
 
   def getTopicList(group: String): List[String] = {
     try {
-      ZkUtils.getChildren(zkClient, s"${ZkUtils.ConsumersPath}/$group/offsets").toList
+      ZkUtils.getChildren(zkClient, s"${ZkUtils.ConsumersPath}/$group/owners").toList
```
Comment (rzukow) [inline src/main/scala/com/quantifind/kafka/OffsetGetter.scala]: Is this breaking backward compatibility with 0.8.1 ?

File: README.md [hunk 2]
```diff
@@ -50,6 +51,7 @@ The arguments are:
 - **retain** how long should points be kept in the DB
 - **dbName** where to store the history (default 'offsetapp')
 - **pluginsArgs** additional arguments used by extensions (see below)
+- **brokerStorage** whether to use the broker storage for offsets (since Kafka 0.8.2)
```
Comment (rzukow) [inline README.md]: IMO this argument should go after port (it is in the group of options defining  how do you fetch offsets)

File: src/main/scala/com/quantifind/kafka/OffsetGetter.scala (hunk 1 above)
Comment (chemicL) [inline src/main/scala/com/quantifind/kafka/OffsetGetter.scala]: Luckily this path is used by both 0.8.1 and 0.8.2 for consumer synchronization and contains the list of topics. Tested on both versions.

File: README.md (hunk 2 above)
Comment (chemicL) [inline README.md]: Agreed, will reorder.

**Comment by chemicL** [comment]
Closing as it's not relevant at this stage any more.

**Pull request closed without merging**